INTITA/Modules/Profile/Cells/ProfileHeaderView.swift

```diff
@@ -20,6 +23,7 @@ class ProfileHeaderView: UITableViewCell {
     @IBOutlet weak var specializationLabel: UILabel!
     
     weak var delegate: ProfileHeaderViewDelegate?
+    weak var dataSourse: ProfileHeaderViewDataSourse?
```

Comment:
I guess there is no need for this row and protocol ProfileHeaderViewDataSourse in this PR, I haven't found callers. Or if it is a template for future tasks, let's write //TODO: inside.

Reply:
I forgot to remove it. Thanks